Dockerfile

```diff
@@ -194,7 +199,7 @@ RUN chmod 755 /*.sh /usr/local/bin/speedtest-cli
 
 VOLUME [ "/opt/fhem" ]
 
-EXPOSE 7072 8083
+EXPOSE 7072 8083 1883
```

Comment:
Extending EXPOSE is not needed here as one cannot be sure that a user is actually using it. It is also not required for the container to work properly as technically this has nothing to do with making ports available outside of the host network. Kindly revert this change.

Reply:
I agree with your comment on EXPOSE.